tasks/internal/push-artifacts-task/tests/test-push-artifacts-only-cdn.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-push-artifacts-only-cdn
+spec:
+  description: |
+    Run the push-artifacts task and only push to CDN
```

Comment:
Can you say why it is only pushing to CDN (cgw values missing in snapshot I think)

Reply:
updated